app/static/js/slcNewProposal.js

```diff
@@ -121,7 +121,7 @@ $(document).ready(function(e) {
   // -----------------------------------------
       $("#instructorTable").on("click", "#remove", function() {
         let closestRow =  $(this).closest("tr")
-        $("#instructorTableNames input[value="+closestRow.data('username')+"]").remove()
+        let username = closestRow.data('username');
```

Comment:
The variable "username" is not used after its instantiation.

Reply:
Fixed it!